src/wren_compiler.c

```diff
@@ -1381,17 +1381,20 @@ typedef enum
 {
   PREC_NONE,
   PREC_LOWEST,
-  PREC_ASSIGNMENT, // =
-  PREC_LOGIC,      // && ||
-  PREC_EQUALITY,   // == !=
-  PREC_IS,         // is
-  PREC_COMPARISON, // < > <= >=
-  PREC_RANGE,      // .. ...
-  PREC_BITWISE,    // | ^ & << >>
-  PREC_TERM,       // + -
-  PREC_FACTOR,     // * / %
-  PREC_UNARY,      // unary - ! ~
-  PREC_CALL,       // . () []
+  PREC_ASSIGNMENT,    // =
+  PREC_LOGIC,         // && ||
+  PREC_BITWISE_OR,    // |
+  PREC_BITWISE_XOR,   // ^
+  PREC_BITWISE_AND,   // &
```

Comment:
I second this. I also think that C precedence aspect was in a grammar quirk.

As for the range-bitwise precedence, I'm disputed, but agree that the common sense would interpret `a & b..c` as `a & (b..c)`.

I'll fix these ones, too.